hack/addons/mindthegap-helm-registry/Dockerfile

```diff
@@ -1,16 +1,9 @@
 ARG MINDTHEGAP_VERSION=v1.14.4
-
 FROM --platform=${BUILDPLATFORM} ghcr.io/mesosphere/mindthegap:${MINDTHEGAP_VERSION} as bundle_builder
 # this gets called by goreleaser so the copy source has to be the path relative to the repo root.
 RUN --mount=source=./hack/addons/mindthegap-helm-registry/repos.yaml,target=/repos.yaml \
     ["/ko-app/mindthegap", "create", "bundle", "--helm-charts-file=/repos.yaml", "--output-file=/tmp/helm-charts.tar"]
 
-FROM --platform=${TARGETPLATFORM} ghcr.io/mesosphere/mindthegap:${MINDTHEGAP_VERSION}
-COPY --from=bundle_builder /tmp/helm-charts.tar /tmp/helm-charts.tar
-VOLUME /certs
-CMD [ "serve", "bundle", \
-      "--bundle=/tmp/helm-charts.tar", \
-      "--listen-port=5000", \
-      "--listen-address=0.0.0.0", \
-      "--tls-private-key-file=/certs/tls.key", \
-      "--tls-cert-file=/certs/tls.crt" ]
+FROM --platform=${TARGETPLATFORM} alpine
+COPY --from=bundle_builder /tmp/helm-charts.tar /charts/
+RUN mv /charts/helm-charts.tar /charts/helm-charts-$(date +%s).tar
```

Comment:
I can't figure out a way to make this unique without using date. I tried passing in .ShortCommit from goreleaser as well as version, but from my understanding the run isn't expanding it for whatever reason.

Reply:
I think this works?
```
diff --git a/.goreleaser.yml b/.goreleaser.yml
index be51152e..f38cf4c7 100644
--- a/.goreleaser.yml
+++ b/.goreleaser.yml
@@ -131,6 +131,7 @@ dockers:
       - "--label=org.opencontainers.image.revision={{.FullCommit}}"
       - '--label=org.opencontainers.image.version=v{{ trimprefix .Version "v" }}'
       - "--label=org.opencontainers.image.source={{.GitURL}}"
+      - '--build-arg=CHARTS_VERSION=v{{ trimprefix .Version "v" }}'
     goarch: arm64
 
 docker_manifests:
diff --git a/hack/addons/mindthegap-helm-registry/Dockerfile b/hack/addons/mindthegap-helm-registry/Dockerfile
index e07b858e..11697c90 100644
--- a/hack/addons/mindthegap-helm-registry/Dockerfile
+++ b/hack/addons/mindthegap-helm-registry/Dockerfile
@@ -6,4 +6,5 @@ RUN --mount=source=./hack/addons/mindthegap-helm-registry/repos.yaml,target=/rep
 
 FROM --platform=${TARGETPLATFORM} alpine
 COPY --from=bundle_builder /tmp/helm-charts.tar /charts/
-RUN mv /charts/helm-charts.tar /charts/helm-charts-$(date +%s).tar
+ARG CHARTS_VERSION
+RUN mv /charts/helm-charts.tar /charts/helm-charts-${CHARTS_VERSION}.tar

```

And it got passed in correctly
```
➜  cluster-api-runtime-extensions-nutanix git:(faiq/copy-charts-to-pvc) ✗ docker run --entrypoint ls -it ghcr.io/nutanix-cloud-native/caren-helm-reg:v0.19.0-dev-arm64 /charts
helm-charts-v0.19.0-dev.tar
```

Reply:
yeah moved it down